### DISCUSSION TITLE
Magick.NET dll fails to convert images from EPS to JPG

### DISCUSSION THREAD

**Discussion by annadurai4003** [ImageMagick/ImageMagick#2794] [Help]
Hi Everyone,

I have imported Magic.NET DLL in Visual studio. It works for TIFF, PNG images. But it doesn't work for EPS image to JPG.

I got the below error while executing for EPS.
`ERROR: Failed ImageMagick.MagickDelegateErrorException: Postscript delegate failed The system cannot find the file specified.
'@ error/ps.c/ReadPSImage/883
   at ImageMagick.MagickImageCollection.NativeMagickImageCollection.ReadFile(MagickSettings settings)
   at ImageMagick.MagickImageCollection.AddImages(String fileName, MagickReadSettings readSettings, Boolean ping)
  `

Do we have options to include GhostScript DLL into the application?

Please anyone help me, how to proceed further. 

Thanks.

**Comment by dlemstra** [comment]
You can find that information in the docs of the project here: https://github.com/dlemstra/Magick.NET/tree/d9f740c2487e7535f60e62d4475dde6f52894319/docs#ghostscript

p.s. The project is located here: https://github.com/dlemstra/Magick.NET.

**Reply by annadurai4003** [reply]
Thank dlemstra. 
Where can I download these files, **gsdll32.dll/gsdl64.dll** and **gswin32c.exe/gswin64c.exe**?

**Reply by dlemstra** [reply]
You can download it here: https://www.ghostscript.com/. For commercial use you need to buy a license from them.

**Reply by annadurai4003** [reply]
Thanks for letting me know.
